assets/css/general.css

```diff
@@ -38,19 +38,33 @@ h2 {
     margin-top: 10px;
 }
 
-/* Utilities */
+.form-group.d-sm-none .translateBtn { /* sass-lint:disable-line class-name-format */
+    margin: 4px;
+    margin-right: 0;
+}
 
-/* sass-lint:disable class-name-format */
-.position-relative {
-    position: relative;
+.translateBtn {
+    background-color: #446e9b !important; /* sass-lint:disable-line no-important */
+    color: #fff !important; /* sass-lint:disable-line no-important */
 }
 
-.d-none {
-    display: none;
+.translateBtn:focus,
+.translateBtn:hover, .translateBtn.focus
+.translateBtn:active, .translateBtn.active {
+    background-color: #385a7f !important; /* sass-lint:disable-line no-important */
+}
+
+/* Monkeypatch for a bug with grid,
+   they don't fit in one line otherwise */
+.dstLangParent {
+    width: calc(50% - 5px);
 }
 
-.d-inline-block {
-    display: inline-block;
+/* Utilities */
+
+/* sass-lint:disable class-name-format */
+.position-relative {
```

Comment:
Does BS4 not already come with an equivalent? I was under the impression that a lot of these utilities could be removed.

Reply:
I removed the unnecessary ones in latest push